src/server/scripts/ScriptLoader.cpp

```diff
@@ -648,8 +651,11 @@ void AddWorldScripts()
     AddSC_npcs_special();
     AddSC_npc_taxi();
     AddSC_achievement_scripts();
-    AddSC_chat_log();
+    AddSC_chat_log(); // location: scripts\World\chat_log.cpp
     AddSC_character_creation();
+    // To avoid duplicate code, we check once /*ONLY*/ if logging is permitted or not.
+    if (sWorld->getBoolConfig(CONFIG_IP_BASED_ACTION_LOGGING))
```

Comment:
I don't know why trinity did this, but AddSC_action_ip_logger(); will never be executed, at least in our code...
The loader always loads first. This causes the pr be useless, it doesn't log in logs_ip_action when a character is deleted, created, etc... We should delete the if and place it somewhere else.